Makefile

```diff
@@ -4,11 +4,18 @@ LIBDIR=${PREFIX}/lib
 INCDIR=${PREFIX}/include
 
 CFLAGS+=-g -Wall -O2 -DDEBUG -fPIC
-LIBS=-lev -levbuffsock -lcurl -ljson-c
+LIBS=-lev -levbuffsock -lcurl
 AR=ar
 AR_FLAGS=rc
 RANLIB=ranlib
 
+ifeq (1, $(findstring 1, $(WITH_JANSSON)))
```

Comment:
I think `findstring` is not needed, just `ifeq (1,$(WITH_JANSSON))` should be fine (or alternatively `ifneq (,$(WITH_JANSSON))`)